### DISCUSSION TITLE
Github copilot vscode extension doesn't respond

### DISCUSSION THREAD

**Discussion by scandar** [community/community#62049] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I'm using the GH copilot vscode extension but some versions don't work at all for me, i had to downgrade to version 1.82.15 since all of the later versions don't work. 

The output console for Github copilot is empty so it's not possible to tell what is wrong. 
Other extensions like Github copilot chat and Github copilot labs work regardless if copilot is working or not. 

My VSCode version is 1.80.1 (Universal)

**Comment by tomerh2001** [comment]
+1

**Comment by OriginalEXE** [comment]
I have the same problem but only in one (large, monorepo) repository, looks like `1.82.15` indeed works as expected, thanks for mentioning it!
